nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -249,7 +217,7 @@ def forward(self, tokens, text_position_ids, attention_mask, labels, prompt_ids=
 
     def setup_optimizer_param_groups(self):
         """ModelPT override. Optimizer will get self._optimizer_param_groups"""
-        self._optimizer_param_groups = _get_params_for_weight_decay_optimization([self.model])
```

Comment:
Is this right?

Reply:
Yeah I just removed the _ when renaming the function.